android/java/res/layout/brave_rewards_panel.xml

```diff
@@ -86,6 +86,7 @@
 
                                     <!-- The 'publisher_favicon_update' substitutes 'publisher_favicon' when not available-->
                                     <ProgressBar
+                                        tools:ignore="TooDeepLayout"
```

Comment:
This will be resolved as part of rewards 2.0 UI changes. good to ignore for now.